app/Models/DelayedJob.php

```diff
@@ -18,17 +18,12 @@ class DelayedJob extends Model
 
     protected $table = 'delayed_jobs';
 
-    protected $fillable = ['uuid', 'status', 'status_code', 'payload', 'entity_type', 'entity_id', 'created_by', 'is_acknowledged', 'name'];
+    protected $fillable = ['uuid', 'status', 'status_code', 'payload', 'metadata', 'created_by', 'is_acknowledged', 'name'];
 
     protected $casts = [
         'uuid' => 'string',
```

Comment:
If you add `'metadata' => 'json'` here, Laravel will handle the serialization for you, and you won't need to `json_encode` or `json_decode` when accessing metadata.

Reply:
changed to array, as that worked.